.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Trigger action for GOV.UK_co-cyber-security-external
+
+on:
+  workflow_dispatch:
+  push:
+    branches:    
+      - 'main'
+    paths:
+      - 'security.txt'
+      - 'thanks.txt'
+  schedule:
+     - cron: "0 2 * * MON-FRI" # Runs at 02:00 UTC
+      workflow_dispatch:
+jobs:
+ deploy-to-co-cyber-security-external:
+  # Trigger makefile for co-cyber-security-external
+    runs-on: ubuntu-latest
+    # Steps represent a sequence of tasks that will be triggered and executed  as part of the job
+    steps:
+       - name: Show workflow_run
+    env:
+      co-cyber-security-external
+      name: Deploy to GOV.UK co-cyber-security-external
+      defaults:
+    run:
+        shell: bash
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+     role-to-assume: arn:aws:iam::576485064911:role/github_actions_role
```

Comment:
yaml formatting issue here

Reply:
Thanks Bart,  let's articulate and Incorporate the above at our meeting slated for  15:30pm this afternoon.
**Review of Pull request for - GitHub-to-AWS keyless authentication**
Tuesday, 20 February · 3:30 – 4:30pm
Time zone: Europe/London
Google Meet joining info
Video call link: https://meet.google.com/eav-pcew-crg
Or dial: ‪(GB) +44 20 3956 2045‬ PIN: ‪175 468 683‬#
More phone numbers: https://tel.meet/eav-pcew-crg?pin=8561866440643
**Regards,
Adebola.**